src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -169,7 +171,23 @@ public HttpRouteCollection Routes
                 };
                 using (logger.BeginScope(scopeState))
                 {
+                    var stopWatch = new Stopwatch();
+                    stopWatch.Start();
                     await Instance.CallAsync(function.Name, arguments, cancellationToken);
+                    stopWatch.Stop();
+
+                    if (request.IsColdStartRequest())
+                    {
+                        var message = new JObject
+                        {
+                            { "requestId", request.GetRequestId() },
```

Comment:
It turns out that currently we aren't actually writing this request ID to the ActivityID column of our table. We can do that later if we need to - will require changes to our event pipeline.

Reply:
Considering also logging out internal request pipeline duration here up to this point. Useful in its own right when looking at these raw logs